packages/vee-validate/src/useForm.ts

```diff
@@ -1281,7 +1294,7 @@ function useFormInitialValues<TValues extends GenericObject>(
   };
 }
 
-function mergeValidationResults(a: ValidationResult, b?: ValidationResult): ValidationResult {
+function mergeValidationResults(a: ValidationResult<never>, b?: ValidationResult<never>): ValidationResult<never> {
```

Comment:
Not a fan of these `never`s, I hardly use the `never` type unless to eliminate a branch in a conditional type. Can you tell me what do you use them for here?